scripts/download-debug-so.sh

```
@@ -0,0 +1,69 @@
+# Variables
+RELEASE_URL="https://github.com/FalkorDB/falkordb/releases/download"
+LIB_PATH="/var/lib/falkordb/bin/falkordb.so"
+DEBUG_LIB_NAME="falkordb-debug-x64.so"
+DOWNLOAD_DIR="/data"
+
+VERSION=""
+# Functions
+get_falkordb_version() {
+  echo "Fetching falkordb version..."
+  local module_info
+  module_info=$(redis-cli -a $(cat /run/secrets/adminpassword) info modules | grep "module:name=graph")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)